client/settings/__tests__/google-play-test-mode-compatibility-notice.test.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import GooglePayTestModeCompatibilityNotice from '../google-play-test-mode-compatibility-notice';
+import { useTestMode, usePaymentRequestEnabledSettings } from 'wcpay/data';
+import WCPaySettingsContext from 'wcpay/settings/wcpay-settings-context';
+
+jest.mock( 'wcpay/data', () => ( {
+	useTestMode: jest.fn(),
+	usePaymentRequestEnabledSettings: jest.fn(),
+} ) );
+
+describe( 'GooglePayTestModeCompatibilityNotice', () => {
+	beforeEach( () => {
+		jest.resetAllMocks();
+	} );
+
+	it( 'does not render when the account is not live', () => {
+		useTestMode.mockReturnValue( [ true ] );
+		usePaymentRequestEnabledSettings.mockReturnValue( [ true ] );
+
+		const { container } = render(
+			<WCPaySettingsContext.Provider
+				value={ { accountStatus: { isLive: false } } }
+			>
+				<GooglePayTestModeCompatibilityNotice />
+			</WCPaySettingsContext.Provider>
+		);
+
+		expect( container.firstChild ).toBeNull();
+	} );
+
+	it( 'does not render when test mode is not active', () => {
+		useTestMode.mockReturnValue( [ false ] );
+		usePaymentRequestEnabledSettings.mockReturnValue( [ true ] );
+
+		const { container } = render(
+			<WCPaySettingsContext.Provider
+				value={ { accountStatus: { isLive: true } } }
+			>
+				<GooglePayTestModeCompatibilityNotice />
+			</WCPaySettingsContext.Provider>
+		);
+
+		expect( container.firstChild ).toBeNull();
+	} );
+
+	it( 'does not render when GooglePlay is not enabled', () => {
```

Comment:
```suggestion
	it( 'does not render when GooglePay is not enabled', () => {
```
Aha, I almost missed this one, until I decided to double-check the changelog. Sorry, lol.

Reply:
Same goes for this filename.